demisto_sdk/commands/common/update_id_set.py

```diff
@@ -1181,12 +1217,24 @@ def re_create_id_set(id_set_path: Optional[str] = DEFAULT_ID_SET_PATH, pack_to_c
     reports_list = []
     widgets_list = []
     mappers_list = []
+    packs_dict: Dict[str, Dict] = {}
 
     pool = Pool(processes=int(cpu_count() * 1.5))
 
     print_color("Starting the creation of the id_set", LOG_COLORS.GREEN)
 
     with click.progressbar(length=len(objects_to_create), label="Progress of id set creation") as progress_bar:
+
+        if 'Packs' in objects_to_create:
+            print_color("\nStarting iteration over Packs", LOG_COLORS.GREEN)
+            for arr in pool.map(partial(process_pack_metadata,
+                                        print_logs=print_logs
+                                        ),
+                                get_pack_metadata_paths(pack_to_create)):
+                packs_dict.update(arr[0])
```

Comment:
I think that it would be better to send it as a dictionary from the helper function.

Reply:
It seems to me that a list must be returned to use the `map` and `partial` functions